v7/import_jekyll/import_jekyll.py

```diff
@@ -213,13 +213,22 @@ def extract_title():
                 name = m.group('name')
             return name
 
+        def extract_image():
+            raw_image = jmetadata.get('image')
+            new_image_path = ''
+            if isinstance(raw_image, str):
+                new_image_path = raw_image.replace('/assets', '/images')
+                new_image_path = new_image_path.replace('posts/images', 'posts')
+            return new_image_path
+
         tags = [x for x in jmetadata.get('tags') or [] if x]
         categories = [x for x in jmetadata.get('categories') or [] if x]
 
         metadata = PageCompiler.default_metadata.copy()
         metadata['title'] = extract_title()
         metadata['slug'] = slugify_file(path)
         metadata['date'] = extract_date()
+        metadata['previewimage'] = extract_image()
```

Comment:
Consider setting `previewimage` conditionally, only if an image is extracted successfully.

Reply:
I should added that check